mungegithub/mungers/blunderbuss.go

```diff
@@ -38,9 +44,14 @@ type BlunderbussConfig struct {
 // BlunderbussMunger will assign issues to users based on the config file
 // provided by --blunderbuss-config.
 type BlunderbussMunger struct {
-	config              *BlunderbussConfig
-	features            *features.Features
 	BlunderbussReassign bool
+	AliasFile           string
+
+	aliasMap      map[string][]string
+	config        *BlunderbussConfig
+	features      *features.Features
+	prevHash      string
```

Comment:
I'd encapsulate the alias related functionality into its own object for maintainability.